database/sql/src/main/scala/cromwell/database/slick/tables/CallCachingHashComponent.scala

```diff
@@ -8,29 +8,27 @@ trait CallCachingHashComponent {
 
   import driver.api._
 
-  class CallCachingHashEntries(tag: Tag) extends Table[CallCachingHashEntry](tag, "CALL_CACHING_HASH") {
+  class CallCachingHashes(tag: Tag) extends Table[CallCachingHashEntry](tag, "CALL_CACHING_HASH") {
```

Comment:
Edit: but revert away. I'll sync the database scala with the database names in some future PR.